jscs/auto-fix.js

```diff
@@ -0,0 +1,14 @@
+// Add support for a --noFix argument to disable JSCS auto-fix.
+
+module.exports = jscsAutoFix;
+
+function jscsAutoFix() {
+  var args = process.argv.slice(2);
+  var noFixArgIndex = args.indexOf('--noFix');
+  var noFix = noFixArgIndex > -1;
+
+  if (noFix) {
+    process.argv.splice(noFixArgIndex + 2, 1);
```

Comment:
Using minimist doesn't cover the need to remove the `--noFix` param before passing it to `fashion-show`. Should I keep this file around to maintain that logic?